### PR TITLE
Update rounds to use timestamps with SSC

### PR DESCRIPTION
**Pull request by amityadav0**
https://discord.com/channels/754286650540032030/754286650540032033/811328347522203738

### CODE REVIEW

File: packages/runtime/test/src/interpreter/opcode-list.ts [hunk 1]
```diff
@@ -2134,14 +2135,12 @@ describe("Teal Opcodes", function () {
...
+      interpreter.runtime.setBlockGlobalData(500, 100);
       const op = new Global(['LatestTimestamp'], 1, interpreter);
       op.execute(stack);
 
       const top = stack.pop();
-      const ts = Math.round((new Date()).getTime() / 1000);
-
-      const diff = Math.abs(ts - Number(top)); // for accuracy
-      assert.isBelow(diff, 10);
+      assert.equal(BigInt('100'), top);
```
Comment (robert-zaremba) [inline packages/runtime/test/src/interpreter/opcode-list.ts:2143]: we can use `100n` instead of `BigInt('100')`

**Review by robert-zaremba — APPROVED** [review]
preapproving - let's also consider `setRoundAndTimestamp` instead of `setBlockGlobalData`

File: packages/runtime/test/src/interpreter/opcode-list.ts (hunk 1 above)
Comment (amityadav0) [reply packages/runtime/test/src/interpreter/opcode-list.ts:2143]: ok

**Comment by amityadav0** [comment]
Updated to `setRoundAndTimestamp`

**Pull request merged**